scripts/generateMetadata.js

```diff
@@ -82,7 +82,9 @@ const getFilesFromDirectory = (directoryPath) => {
     .filter((data) => data)
     .reduce((previousValue, currentValue) => ({ ...previousValue, ...currentValue }), {});
 
-  const fileContent = `const metadata = ${JSON.stringify(formattedData)}; export default metadata;`;
+  const fileContent = `// This file is autogenerated, please don't modify it directly
```

Comment:
Can you add a note directing people to this file?

Reply:
Done